src/SignalR/server/Core/src/HubConnectionHandler.cs

```diff
@@ -223,8 +223,6 @@ private async Task DispatchMessagesAsync(HubConnectionContext connection)
                 var result = await input.ReadAsync();
                 var buffer = result.Buffer;
 
-                connection.ResetClientTimeout();
```

Comment:
? This moved?

Reply:
It's no longer needed with how we changed timeouts